run.py

```diff
@@ -189,20 +180,36 @@ def main(argv):
     dev = ''
     container = 'docker'
     try:
-        opts, args = getopt.getopt(argv, "hd:c:r:p:b:l", ["dev=", "container=", "report", "plot=","benchmark=", "list_benchmarks"])
+        opts, args = getopt.getopt(
+            argv,
+            "hd:c:r:p:b:lo:",
```

Comment:
I think this line should be
```suggestion
            "hd:c:r:p:b:l:o",
```

Reply:
l does not take an argument ?

Reply:
Okay than this is resolved :)